tests/vmi_multus_test.go

```diff
@@ -995,6 +1004,46 @@ var _ = Describe("[Serial]Macvtap", func() {
 			})
 		})
 	})
+
+	Context("VMI migration", func() {
+		var macvtapVMI *v1.VirtualMachineInstance
+
+		cleanupVMI := func(vmi *v1.VirtualMachineInstance) error {
+			err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Delete(macvtapVMI.GetName(), &metav1.DeleteOptions{})
+			err = wait.PollImmediate(time.Second, 2*time.Minute, func() (done bool, err error) {
+				_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(macvtapVMI.GetName(), &metav1.GetOptions{})
+				return err != nil && errors.IsNotFound(err), nil
+			})
+			return err
+		}
+
+		BeforeEach(func() {
+			nodes := tests.GetAllSchedulableNodes(virtClient)
+			Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
+
+			if len(nodes.Items) < 2 {
+				Skip("Migration tests require at least 2 nodes")
```

Comment:
Shouldn't you verify the migration feature gate is [enabled](https://github.com/kubevirt/kubevirt/blob/115c51ff8a5bb365a6f0874f9c91b984ca85411d/tests/migration_test.go#L147) as well?

Reply:
Good point. No clue - I didn't even know it had a feat. gate.

I'll add a check for it.

Reply:
Also, the migration test have another pre-step related to [taints](https://github.com/kubevirt/kubevirt/blob/115c51ff8a5bb365a6f0874f9c91b984ca85411d/tests/migration_test.go#L160).
I have no idea why this step is required, please check if it required here as well.

Reply:
I have no idea what those are, and all I can say is that after the migration - which succeeds - there's a new VMI running on a different node, having a different UID (that is what the test checks). As such, I think that is not required.

I'd avoid introducing things I have no clue about, and I don't have buffer space to dig onto that. 

If you say we need to figure this out, I'd prefer to remove migration from the scope of this PR, and address that use case on a follow-up PR.